src/mmw/js/src/app.js

```diff
@@ -155,6 +155,57 @@ function RestAPI() {
     };
 }
 
+function adjustCompareViewBeforeITSIScreenshot() {
+    toggleCompareViewForITSIScreenshot(true);
+}
+
+function adjustCompareViewAfterITSIScreenshot() {
+    toggleCompareViewForITSIScreenshot(false);
+}
+
+function toggleCompareViewForITSIScreenshot(adjustForScreenshot) {
+    var itsiCompareDialog = 'itsi-compare-dialog',
+        itsiCompareModal = 'itsi-compare-modal',
+        itsiCompareRow = 'itsi-compare-row',
+        compareDialog = '#compare-new-dialog',
+        compareModalContent = '.compare-modal-content',
+        compareCloseButton = '.compare-close',
+        compareChartButton = '#compare-input-button-chart',
+        compareTableButton = '#compare-input-button-table',
+        compareChartRow = '.compare-chart-row',
+        compareTableRow = '.compare-table-row',
+        compareScenariosRow = '.compare-scenarios',
+        compareMapsRow = '.compare-scenario-row-content';
+
+    if (adjustForScreenshot) {
+        $(compareDialog).addClass(itsiCompareDialog);
```

Comment:
Consider doing the `$(...)` in the `var` declaration above rather than down here, to avoid repeated DOM scans for the same things.

Reply:
Good point. I tried this out a bit & wasn't fully able to get it to work, but I think we should try to make that adjustment when/if we update this to account for the scroll behavior.